x-pack/plugins/infra/server/lib/adapters/fields/framework_fields_adapter.ts

```diff
@@ -36,91 +36,15 @@ export class FrameworkFieldsAdapter implements FieldsAdapter {
 
   public async getIndexFields(
     requestContext: RequestHandlerContext,
-    indices: string,
-    timefield: string
+    indices: string
   ): Promise<IndexFieldDescriptor[]> {
     const indexPatternsService = this.framework.getIndexPatternsService(requestContext);
     const response = await indexPatternsService.getFieldsForWildcard({
       pattern: indices,
     });
-    const { dataSets, modules } = await this.getDataSetsAndModules(
-      requestContext,
-      indices,
-      timefield
-    );
-    const allowedList = modules.reduce(
-      (acc, name) => uniq([...acc, ...getAllowedListForPrefix(name)]),
-      [] as string[]
-    );
-    const dataSetsWithAllowedList = [...allowedList, ...dataSets];
     return response.map(field => ({
       ...field,
-      displayable: dataSetsWithAllowedList.some(name => startsWith(field.name, name)),
+      displayable: true,
     }));
   }
-
-  private async getDataSetsAndModules(
-    requestContext: RequestHandlerContext,
-    indices: string,
-    timefield: string
-  ): Promise<{ dataSets: string[]; modules: string[] }> {
-    const params = {
-      index: indices,
-      allowNoIndices: true,
-      ignoreUnavailable: true,
-      body: {
-        size: 0,
-        query: {
-          bool: {
-            filter: [
-              {
-                range: {
-                  [timefield]: {
-                    gte: 'now-24h',
-                    lte: 'now',
-                  },
-                },
-              },
-            ],
-          },
-        },
-        aggs: {
-          datasets: {
-            composite: {
-              sources: [
-                {
-                  dataset: {
-                    terms: {
-                      field: 'event.dataset',
-                    },
-                  },
-                },
-              ],
-            },
-          },
-        },
-      },
-    };
-
-    const bucketSelector = (response: InfraDatabaseSearchResponse<{}, DataSetResponse>) =>
-      (response.aggregations && response.aggregations.datasets.buckets) || [];
-    const handleAfterKey = createAfterKeyHandler(
-      'body.aggs.datasets.composite.after',
-      input => input?.aggregations?.datasets?.after_key
-    );
-
-    const buckets = await getAllCompositeData<DataSetResponse, Bucket>(
-      this.framework,
-      requestContext,
-      params,
-      bucketSelector,
-      handleAfterKey
-    );
-    const dataSets = buckets.map(bucket => bucket.key.dataset);
-    const modules = dataSets.reduce((acc, dataset) => {
-      const module = first(dataset.split(/\./));
```

Comment:
The compiler complains about these lodash functions still being imported after the removal of their usage.